src/proto_bcd/findings/finding_category.py

```diff
@@ -46,6 +46,8 @@ class FindingCategory(enum.Enum):
     RESOURCE_DEFINITION_REMOVAL = 24
     RESOURCE_PATTERN_REMOVAL = 25
     RESOURCE_PATTERN_ADDITION = 26
+    # Messages to files mapping
```

Comment:
nitty nit: s/Messages to files/Messages-to-files/

Reply:
Done!